src/substrait/textplan/Location.cpp

```diff
@@ -6,7 +6,7 @@
 
 namespace io::substrait::textplan {
 
-const Location Location::kUnknownLocation(
+constexpr const Location Location::kUnknownLocation(
```

Comment:
```suggestion
constexpr Location Location::kUnknownLocation(
```

I could be wrong but I think, in this case, `constexpr const` is redundant.

Reply:
Done.  Only needed for pointers apparently.